### DISCUSSION TITLE
How to advocate for moving beyond "greenwashing"

### DISCUSSION THREAD

**Discussion by scottrigby** [cncf/tag-env-sustainability#30] [General]
discussion from meeting today:
- it seems everyone agrees we do not want to do this, deliberately or accidentally
- how to clearly send this message from the wg/tag
    - [Scott & Cara] Craft what our intentions are to make sure we are not greenwashing - create PR on existing Charter
    - add to glossary?
- [Leo] how to avoid "policing"

Any other thoughts on this?

**Comment by leonardpahlke** [comment]
I agree that this issue is still a problem. But there are some considerations (that I can see, atm):

1. how does this relate to Cloud Native (do we see greenwashing in the cloud native space, or do we just want to advocate about sustainability in general?)
2. where mention greenwashing in the charter? (@scottrigby you said as an option under goal / non-goal, sounds good to me).
3. what could be the action items from this?
      To me, this sounds like a topic we could discuss in a blog post, that said, I don't see this turning into a project (maybe later). 

I think it's a good idea to explicitly mention this in the charter :+1:  :slightly_smiling_face:

**Reply by catblade** [reply]
I did call out greenwashing in the new TAG proposal.  Please review the document in this branch:  https://github.com/catblade/wg-env-sustainability/blob/TAG-env-sustainability-proposal/documentation/EnvironmentalSustainabilityTAGProposal.md

This is document is also noted on the mailing list.

**Comment by mkorbi** [comment]
Need to reopen this topic:
https://github.com/cncf/tag-env-sustainability/issues/46
